src/pages/Home/index.jsx

```diff
@@ -47,20 +54,24 @@ const Home = () => {
   const handleSubmit = async (event) => {
     if (apiCall) return;
     event.preventDefault()
-    setApiCall(true);
-
     setUsernameError(false)
     setPasswordError(false)
 
-    if (username == '') {
```

Comment:
I think we should not allow capital letters here

Reply:
Done